ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -654,8 +655,8 @@ instance
         len' <- CBOR.decodeListLen
         tag' <- CBOR.decodeWord8
         when
-          (len /= 1 + len' || tag /= tag')
-          (fail $ "decode @ApplyMapKind " <> show (smk, len', tag'))
+          (1 + len /= len' || tag /= tag')
```

Comment:
Could we comment on why we check this assertion here?

Reply:
Done!